packages/scss/src/components/_button.scss

```diff
@@ -297,15 +296,56 @@
 		position: relative;
 
 		&:first-child {
```

Comment:
Maybe we could have these :first-child and :last-child moved to a `> .button` selector, to avoid having this style bleeding over the .button-group-item